frontend-react/src/contexts/SessionContext.test.tsx

```diff
@@ -1,47 +1,69 @@
-import { render, screen } from "@testing-library/react";
-import { useContext } from "react";
+import { screen } from "@testing-library/react";
 
 import { MemberType } from "../hooks/UseOktaMemberships";
+import { mockUseOktaMemberships } from "../hooks/__mocks__/UseOktaMemberships";
+import { renderWithSession } from "../utils/CustomRenderUtils";
 
-import { SessionContext } from "./SessionContext";
+import { useSessionContext } from "./SessionContext";
 
-const TestComponent = () => {
```

Comment:
What the old version of this component did is A) my fault and B) very annoying. It's also covered in useOktaMemberships tests. The provider's only logic is the admin hard check. The diff will show the updated component with a comment as to why it even needs to be there. If anyone knows a better way to test context state without a component, let me know!

Reply:
yeah, the context is really only being used to create the provider, which only does anything when interacting with a dom of some sort, so you kind of have to render something.

i do think you could do this sort of test against the provider, rather than the hook, though. I think this is fine, but maybe something to look at more and try to standardize in testing conventions soon.

Reply:
What do you mean "you could do this sort of test against the provider, rather than the hook"?